static/home/css/style.css

```diff
@@ -702,35 +699,34 @@ a:hover{
 	justify-self: center;
 }
 
-.contact-details-div {
+.don-box3 .contact-details-div {
 	display: flex;
 	align-items: center;
 	justify-content: space-between;
 	margin-bottom: 12px;
 	
 }
 
-.details-1 {
+.details-1, .details-2 {
 	text-align: center;
     max-width: 150px;
     margin-bottom: 12px;
 	margin-top: 12px;
+	
+}
```

Comment:
I think you didn't understand well what I suggested earlier.

* As both the classes have some common properties, you can put them in a single class name `.contact-details` (put this class in both the elements).
* Now for the different margins of two divs, put a class name `.contact-details-left` in the left div and class name `.contact-details-right` in the right div and apply corresponding margins to both the classes.
* Add `.don-box3` before both the classes in CSS, so that these properties only apply to `.contact-details`, `.contact-details-left` and `contact-details-right` classes inside the `.don-box3` and not anywhere else if we use the same class name somewhere else. Ex. `.don-box3 .contact-details`.